app/services/subscribe_service.rb

```diff
@@ -43,7 +43,7 @@ def subscription_params
   end
 
   def some_local_account
-    @some_local_account ||= Account.local.where(suspended: false).first
+    @some_local_account ||= Account.representative || Account.local.where(suspended: false).first
```

Comment:
no reason to use Account.representative here.

Reply:
Certainly, that's right.